localisation/00_zuwc_united_localisation.csv

```diff
@@ -558,7 +558,7 @@ EVTOPTA_WCTRD_310;Truly a glorious death!;;Ein wahrhaft ehrenwerter Tod!;;;;;;;;
 EVTDESC_WCTRD_311;You see the last moments of your life. You died dishonourably, brought low by the blow of a dagger. This vision frightens you - who would assassinate you? You will have a life full of fear and doubts about your destiny.;;Ihr erblickt die letzten Augenblicke Eures Lebens. Ihr sterbt einen ehrlosen Tod, niedergestreckt durch einen Dolch im R�cken. Diese Vision jagt Euch Angst ein - wer w�rde Euch ermorden wollen? Ein Leben voller Angst und Zweifel liegt vor Euch.;;;;;;;;;;;x
 EVTOPTA_WCTRD_311;So� that's how it will be...;;Das erwartet mich also...;;;;;;;;;;;x
 EVTDESC_WCTRD_312;You see the last moments of your life. You die in the heat of battle, taking many enemies with you as you let loose one last powerful roar that echoes through the forest. There is no doubt that you will have a life full of blood-fury and battles.;;Ihr erblickt die letzten Augenblicke Eures Lebens. Ihr sterbt in der Hitze des Gefechts, viele Eurer Feinde mit einem letzten w�tenden Schrei mitrei�end, der durch den Wald widerhallt. Zweifellos erwartet Euch ein Leben voller Wut und K�mpfe.;;;;;;;;;;;x
-EVTDESC_WCUND_300_another;The prisoner you chose is prepared for the transformation. Wielding daggers, your acolytes commit ritual murder to carry out the process. [From.GetSheHeCap] almost left this world, but you brought [From.GetHerHis] soul back using dark magic.;;Der Gefangene, den Ihr f�r die Verwandlung ausgew�hlt habt, wurde vorbereitet. Mit Dolchen in den H�nden f�hrten Eure Akolyten den rituellen Mord aus. [From.GetSheHeCap] hat diese Welt bereits fast verlassen, aber Ihr bringt [From.GetHerHis]e Seele mit Hilfe der dunklen Magie zur�ck.;;;;;;;;;;;x
+EVTDESC_WCUND_300_another;The prisoner of your choice has been prepared for the transformation ritual. Your faithful acolytes gather around the unholy altar, holding their daggers and waiting for your further orders.;;;;;;;;;;;;;x
```

Comment:
add german: "Euer Gefangener wurde auf das Transformationsritual vorbereitet. Eure treuen Akolythen versammeln sich um den unheiligen Altar, halten ihre Dolche fest in Händen und warten auf Eure weiteren Befehle."